src/Umbraco.PublishedCache.NuCache/Persistence/NuCacheContentRepository.cs

```diff
@@ -1000,7 +1025,7 @@ private IEnumerable<ContentSourceDto> GetContentNodeDtos(Sql<ISqlContext> sql, G
         {
             // Use a more efficient COUNT query
             Sql<ISqlContext>? sqlCountQuery = SqlContentSourcesCount()
-                .Append(SqlObjectTypeNotTrashed(SqlContext, nodeObjectType));
+                .Append(SqlObjectTypeNotTrashed(SqlContext, Constants.ObjectTypes.ContentItem));
```

Comment:
Shouldn't this be `Constants.ObjectTypes.Document`?

Reply:
You are completely right.. :)